app/assets/stylesheets/webapp/webviews.scss

```diff
@@ -294,6 +294,14 @@ body.webapp {
     text-decoration: underline;
     color: #0C75E1;
   }
+  #pages {
```

Comment:
this is unrelated to this PR, but I've always found it weird that we have a `webapp` dir in our assets or a file named `webviews` since this whole app is a web app at the moment. maybe something to rename in the near future. created a card for this: https://trello.com/c/i0COT88J